build.sbt

```diff
@@ -524,6 +523,10 @@ lazy val rspaceBench = (project in file("rspace-bench"))
   .enablePlugins(JmhPlugin)
   .dependsOn(rspace % "test->test", rholang % "test->test", models % "test->test")
 
+lazy val secp256k1 =
+  (project in file("secp256k1"))
+    .settings(commonSettings, libraryDependencies ++= (commonDependencies :+ guava))
+
 lazy val rchain = (project in file("."))
   .settings(commonSettings: _*)
   .aggregate(
```

Comment:
The new _secp256k1_ project should also be added to aggregate _rchain_ project definition. Then tests will be executed on CI as part of _REMINDER_.
https://github.com/rchain/rchain/runs/7821254047

Reply:
https://github.com/rchain/rchain/pull/3800/commits/877ad0cd2b68a0767b7cc22f50fb63a2ec16c75a#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R567